docs/tutorial/gatsby-image-tutorial

```diff
@@ -0,0 +1,275 @@
+
+```
+npm install gatsby-image gatsby-transformer-sharp gatsby-plugin-sharp
+```
+
+Once that's done, you want to add the newly installed plugins to you `gatsby-config.js` file. The config file ends up looking like this (other plugins already in use have been removed from this snippet for simplicity). Note that once `gatsby-image` has been installed it does not need to be included in the gatsby-config.js file.
+
+```jsx
+plugins: [`gatsby-transformer-sharp`, `gatsby-plugin-sharp`]
+```
+
+### Gatsby-image Configuration
+
+Now you're set up to use `gatsby-image`. The first thing to do determine where your image files are located. In this example they're in `src/data`.
+
+If you haven't already, make sure that your project is set up to see content inside that directory. That means doing two things. If you created your project using `gatsby new <name>` then the first step is already done for you.
+
+```
+npm install gatsby-source-filesystem
+```
+
+The second step is to make sure your `gatsby-config.js` file specifies the correct folder. In this example it would look like this.
+
+```jsx
+{ resolve: `gatsby-source-filesystem`,
+    options: { path: `./src/data/` }
+}
+```
+
+Now you're ready to go.
+
+### Single Image Query
+
+Your first option is to use `graphql` to query an image file directly. You can include the relative path to the image file and determine how you want `gatsby-image` to process the file.
+
+```jsx
+export const query = graphql`
+  query {
+    file(relativePath: { eq: "headers/headshot.jpg" }) {
+      childImageSharp {
+        fixed(width: 125, height: 125) {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
+```
+
+There are a couple of things to note here. You might expect the relative path to be relative to the file the code sits in, in this case that's index.js. However, that doesn't work. The relative path is actually based on the line of code you put in the `gatsby-source-filesystem` config, which points to `src/data`.
+
+Another thing to note is that this query is using `GatsbyImageSharpFixed`. Another option is `GatsbyImageSharpFluid` which will not take specific dimension specifications. In `gatsby-image`, fluid images are meant for images that don’t have a finite size depending on the screen, where as other images are fixed.
+
+The query will return a processed image at the highest depth. The result will be automatically passed into the component and attached to `data`. Using JSX you can display the image.
+
+Start by importing the image tag provided by `gatsby-image`.
+
+```jsx
+import Img from "gatsby-image"
+```
+
+Now you can use it. Note that the key for pointing to the image corresponds to the way in which the image was processed. In this example that is `fixed`.
+
+```jsx
+<Img
+  className="headshot"
+  fixed={data.file.childImageSharp.fixed}
+  alt="headshot"
+/>
+```
```

Comment:
I feel like a full component or page example showing the query and the component usage here would help tell a bit more of the story as well as set up the `useStaticQuery` example later on. Perhaps it could say something like "here's the query and usage all put together"